ducktape/cluster/vagrant.py

```diff
@@ -76,8 +88,21 @@ def __init__(self):
         # go through and find fully qualified domain name for each node
         # this makes it possible to not require write access to /etc/hosts on the test driver machine
         is_aws = self._is_aws()
+
+        externally_routable_ip = {}
         for node_account in self.available_nodes:
             node_account.externally_routable_ip = self._externally_routable_ip(is_aws, node_account)
+            externally_routable_ip[str(node_account)] = node_account.externally_routable_ip
+
+        # cache cluster info by writing info to a local file in JSON format.
+        cluster_json["is_aws"] = is_aws
+        cluster_json["externally_routable_ip"] = externally_routable_ip
+
+        # Do not create a new file and cache cluster info in unit test
+        try:
+            json.dump(cluster_json, open(ConsoleConfig.CLUSTER_INFO_FILE, "w"))
+        except:
```

Comment:
Can we catch a specific exception type? Catching all exceptions like this is error prone because if something changes and you start throwing some *other* exception, you won't be able to see the issue.